lib/core/include/irods/rodsError.h

```diff
@@ -92,6 +125,28 @@ int freeRErrorContent(struct ErrorStack *myError);
 /// \retval USER__NULL_INPUT_ERR if rError is NULL
 int printErrorStack(struct ErrorStack *rError);
 
+/// \brief Print the contents of the provided ErrorStack to given file
+///
+/// \parblock
+/// The output takes the following form:
+///
+/// Level 0: <error message>
+/// Level 1: <error message>
+/// ...
+/// Level 99: <error message>
+///
+/// If the error status for a particular ErrorMessage is STDOUT_STATUS, "Level <int>: " is not printed.
+/// \endparblock
+///
+/// \param[in,out] stream the stream where the ErrorStack will be printed to
+/// \param[in,out] rError the ErrorStack which is to have its contents printed
+///
+/// \returns error code
+/// \retval 0 always
+/// \retval USER__NULL_INPUT_ERR if rError is NULL
+/// \retval FILE_WRITE_ERROR if std::fprintf fails
+int print_error_stack_to_file(FILE* stream, const struct ErrorStack* rError);
```

Comment:
Consider swapping the order of the parameters.

The other functions in this file take `ErrorStack` as the first parameter.

Reply:
I put the `FILE` first to look like some c functions such as `fprintf`. However, there are other c functions like `fputs` where the data goes first followed by the `FILE` pointer. Which one do you think makes more sense?

Reply:
Yeah, C isn't very consistent with ordering of parameters.

Let's swap them.